workspace/src/main/kotlin/com/cosmotech/workspace/azure/WorkspaceServiceImpl.kt

```diff
@@ -381,7 +385,8 @@ internal class WorkspaceServiceImpl(
   ): WorkspaceAccessControl {
     val workspace = findWorkspaceByIdNoSecurity(organizationId, workspaceId)
     csmRbac.verify(workspace.getRbac(), PERMISSION_WRITE_SECURITY)
-    csmRbac.getAccessControl(workspace.getRbac(), identityId)
+    csmRbac.getAccessControlOrThrow(
+        workspace.getRbac(), identityId, "User '$identityId' not found in workspace $workspaceId")
```

Comment:
Following the dev review on cosmotech-api-common, use the correct method `(verify|check)Rights`